### PR TITLE
Review reference module resources

### PR DESCRIPTION
**Pull request by asteflova**
In this pull request, I reviewed the description of reference modules. The new version matches the structure used for explaining the other modules and for assemblies: a definition first, followed by writing guidelines, and then links to useful resources.

The reference module guidelines were the trickiest for me to review because I find reference modules the most difficult to explain, but I did my best. The updated version is based on the guidelines we already had in the manual and a few conversations I've had with people over the past few months.

### CODE REVIEW

**Comment by rkratky** [comment]
Ack

File: modular-docs-manual/content/topics/module_guidelines-reference.adoc [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
+The introduction to a reference module is a single, concise paragraph that provides a short overview of the module. A short description makes the module more usable because users can quickly determine whether the reference is useful without having to read the entire module.
 
-* If you have a large volume of the same type of information to document, figure out a consistent structure into which the information details can fit, and then doument each logical unit of information as one reference module. For example, think of man pages, which document very different information details, but which still use consistent titles and formats to present those details in a consistent information structure.
+[discrete]
+== Writing the Reference
 
-== Additional Resources
+A reference module has a very regimented structure, often in the form of a list or a table. A well-organized reference module enables users to scan it quickly to find the details they want.
```
Comment (rkratky) [inline modular-docs-manual/content/topics/module_guidelines-reference.adoc]: "regimented" structure is not simple English. Perhaps we could use "strict" or "firm"?

File: modular-docs-manual/content/topics/module_guidelines-reference.adoc [hunk 2]
```diff
@@ -1,15 +1,21 @@
...
 
-Download the reference module link:https://https://raw.githubusercontent.com/redhat-documentation/modular-docs/master/modular-docs-manual/files/TEMPLATE_REFERENCE_reference-template-and-guidelines.adoc[template] for new projects.
+To make the reference data easier to scan, organize it in a logical order (such as alphabetically) or as a table. AsciiDoc markup to consider for reference data:
+
+* link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#lists[Lists] (unordered, labeled)
+* link:http://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#tables[Tables]
+
+If you have a large volume of the same type of information to document, figure out a consistent structure into which the information details can fit, and then document each logical unit of information as one reference module. For example, think of man pages, which document very different information details, but which still use consistent titles and formats to present those details in a consistent information structure.
```
Comment (rkratky) [inline modular-docs-manual/content/topics/module_guidelines-reference.adoc]: I'd suggest not starting with "if". Instead: "To document a large volume..."

Comment (rkratky) [inline modular-docs-manual/content/topics/module_guidelines-reference.adoc]: "figure out" is a phrasal verb. Could we substitute with "establish", "choose", or just "use"?

Comment (rkratky) [inline modular-docs-manual/content/topics/module_guidelines-reference.adoc]: "consistent" used 3 times in this para. I'd suggest:

- leave out the first
- leave the second as it is
- substitute the third with "uniform", "unchanging", or "persistent"

**Comment by bhardesty** [comment]
Looks good, @asteflova

File: modular-docs-manual/content/topics/module_guidelines-reference.adoc (hunk 1 above)
Comment (asteflova) [reply modular-docs-manual/content/topics/module_guidelines-reference.adoc]: Fixed

File: modular-docs-manual/content/topics/module_guidelines-reference.adoc (hunk 2 above)
Comment (asteflova) [reply modular-docs-manual/content/topics/module_guidelines-reference.adoc]: Fixed

Comment (asteflova) [reply modular-docs-manual/content/topics/module_guidelines-reference.adoc]: Fixed

Comment (asteflova) [reply modular-docs-manual/content/topics/module_guidelines-reference.adoc]: Understood, but I still prefer "If" in these situations. "If" sends a clear signal that what follows is a decision (yes = read on, no = feel free to skip). The decision is still there with "To...", but it's less clear.

**Comment by ritz303** [comment]
lgtm, ack.

**Pull request merged**